### ISSUE TITLE
Final Review

### ISSUE THREAD

**Issue by asheelamagwili** [introdsci/DataScience-bolson10#7]
# Summary

* This project (1) investigates the financial data of public school districts and its affect on performance. (2) Brenndan's found that the districts in the sample had similar funding per student, but vary on how successful those students are in terms of graduation rate or GPA. (3) The next steps planned are to better calculate what it actually means to be a "better performing" school by taking GPA, graduation rate, and level of education into consideration.*

# Data Preparation

*In the first deliverable the tables developed are (1) state revenue data and school districts. Brenndan chose to focus on the fall attendance, transportation, and school lunch by district as well as state revenue by county. In the second deliverable a new dataset is introduced that informs us on district rankings in 2017. (2) The portfolio demonstrates that the data is organized neatly in each table and easy to read. I can properly comprehend what each dataset contains and what it is telling us as readers. the visuals of how the data looks it's cleaned form in relation to each other helps well with understanding what we have. (3) The portfolio demonstrates cleaned data that is easy to read and understand. *

# Modeling

* (1) The predictive model built was not clearly stated but by looking at the R code it appears to be Linear Regression, maybe stating this specifically would help readers understand better. The dependent variable used as a predictor is **State Revenue**. (2) The portfolio clearly describes that the model is being used to predict School District's Ranking. (3) The models are clearly and accurately interpreted in the summary. It is easy to follow along as well as comprehend what they are referring to in the visuals. *

# Validation

* (1)The models are cross-validated using testing and training sets. (2) The accuracy of the cross-validation is clearly shown using visuals as well as in the comments. *

# R Proficiency

* **Strengths - ** The content of the portfolio is clear and concise. The flow of information is easy to follow because of the clear objective. Referring to visuals in the explanation was also very understanding. The results are easy to interpret because they were explained well. All appropriate R techniques were implemented.

**Weaknesses - ** The markdown formatting of the portfolio was a bit hard to read. Paragraph after paragraph was difficult to follow along with. Maybe taking advantage of the markdown formatting would make the look/aesthetic of your portfolio easier to read. *

# Communication

* The portfolio had ample amounts of detail that explained the content and  objectives of the research. There was not a point where I did not understand what was being said, it was well communicated in plain English. There are at times where I feel I missed a step during the Modeling part, maybe adding a little more explanation of what each code chunk does to help others understand your goal. *

# Critical Thinking

* Throughout the project the objective was clear and was consistently referred to, however in the third deliverable I wasn't able to spot where operationalization was clearly stated. A possibility to fix this could be to add a section where examples of this could be shown. It is referred to in the last few paragraphs but clearly stating them would help as well. A clear reference to how this research could help society is stated in the last deliverable. Some possible unintended consequences that the author has not discussed would be that not all states are equal and not all school are equal.*

**Comment by bolson10** [comment]
# Data Preparation and Modeling (*your score* out of 20%)
20% 

All data is clean and tidy, and the models were properly created/tested


# Validation and Operationalization (*your score* out of 20%)
19%

Cross-validation is used, and explains the reason for doing so. The future is discussed in detail if this research was to continue.

# R Proficiency (*your score* out of 20%)
20%

No "awkward" code, and everything is reusable and relevant. 


# Communication (*your score* out of 20%)
18%

Most of it is very easy to read and describes what is being done. Some parts are a little dull.


# Critical Thinking (*your score* out of 20%)
19%

Analysis of models/graphs contains meaningful insights and raises relevant questions.
Insights about the implications and consequences are referenced.
